pandas/_testing/asserters.py

```diff
@@ -1066,25 +1070,25 @@ def assert_series_equal(
 
 # This could be refactored to use the NDFrame.equals method
 def assert_frame_equal(
-    left,
-    right,
-    check_dtype=True,
-    check_index_type="equiv",
-    check_column_type="equiv",
-    check_frame_type=True,
-    check_less_precise=no_default,
-    check_names=True,
-    by_blocks=False,
-    check_exact=False,
-    check_datetimelike_compat=False,
-    check_categorical=True,
-    check_like=False,
-    check_freq=True,
-    check_flags=True,
-    rtol=1.0e-5,
-    atol=1.0e-8,
-    obj="DataFrame",
-):
+    left: DataFrame,
+    right: DataFrame,
+    check_dtype: Union[bool, str] = True,
```

Comment:
just bool

Reply:
on L128 we call `assert_frame_equal` with `check_dtype=check_dtype` and the default value of `check_dtype` in that context is `"equiv"`. If we don't type it is as `Union[bool, str]` we get the following error:  

```python
error: Argument "check_dtype" to "assert_frame_equal" has incompatible type "Union[bool, str]"; expected "bool"  [arg-type]
```

Reply:
could use Literal["equiv"] here instead of str for clarity.